src/main/groovy/org/transmartproject/batch/highdim/jobparams/StandardHighDimDataParametersModule.groovy

```diff
@@ -52,6 +50,14 @@ class StandardHighDimDataParametersModule
 
     void munge(ExternalJobParametersInternalInterface ejp)
             throws InvalidParametersFileException {
+        if (ejp[LOG_BASE] == null) {
+            ejp[LOG_BASE] = '2'
+        }
+
+        if (ejp[DATA_TYPE] == null) {
+            ejp[DATA_TYPE] = 'R'
+        }
```

Comment:
The change was made to make `DATA_TYPE` specification optional by user. User had to specify `DATA_TYPE=R` explicitly before.